highfidelity/hifi
# Case 20887: Move FST joint property handling to the model preparation step

https://highfidelity.manuscript.com/f/cases/20887/Move-FST-joint-association-to-Model-Preparation-Step

This PR moves some of the processing of the FST file out of FBXSerializer and into the model prep step, with a focus on joints. It also cleans up some unused properties I found while figuring out what to move.

The following FST properties now remain in FBXSerializer: "bs" (for blendshapes), "materialMap", and properties related to the rescaling of the model.

## TEST PLAN
- Verify animation for arbitrary test avatars (walking, emotes) is the same as in master, and the avatar's default resting pose is the same.
- Verify avatar packager still works
